### PR TITLE
[bitnami/apache] add path to http probes

### PR DESCRIPTION
**Pull request by whall89**
**Description of the change**

I have added the option to allow the pod health checks(liveness and readiness probes) be pointed at other paths and ports different from the defaults, "/" and "http".

**Benefits**

This allows the pod to point to a specific path and port on apache weather it is an html file, other web app or in our case "/server-status" the built-in apache health check.

**Possible drawbacks**

if the path and port are changed from default and arent configured properly in the apache vhost, config or web-app it could cause an error.

**Applicable issues**

N/A

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/apache/values.yaml [hunk 1]
```diff
@@ -121,13 +121,17 @@ resources:
...
+  port: http
   initialDelaySeconds: 180
   periodSeconds: 20
   timeoutSeconds: 5
   failureThreshold: 6
   successThreshold: 1
 readinessProbe:
   enabled: true
+  path: "/"
+  port: http
```
Comment (juan131) [inline bitnami/apache/values.yaml]: These new parameters should be listed in the README.md

File: bitnami/apache/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: apache
-version: 7.2.14
+version: 7.2.15
```
Comment (juan131) [inline bitnami/apache/Chart.yaml]: Bump the minor version instead.

File: bitnami/apache/templates/deployment.yaml [hunk 3]
```diff
@@ -90,8 +90,8 @@ spec:
           {{- if .Values.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: /
-              port: http
+              path: {{ .Values.readinessProbe.path }}
+              port: {{ .Values.readinessProbe.port }}
```
Comment (juan131) [inline bitnami/apache/templates/deployment.yaml]: I understand that you want to modify the "path" to check, but why would you change the port? I mean, no matter what port you chose at `.Values.service.port`, the port used in the container will be named "http" (check line 75)

File: bitnami/apache/values.yaml (hunk 1 above)
Comment (whall89) [reply bitnami/apache/values.yaml]: new path parameter added as well as readinessProbe.enabled and livenessProbe.enabled

File: bitnami/apache/Chart.yaml (hunk 2 above)
Comment (whall89) [reply bitnami/apache/Chart.yaml]: removed patch version update and updated minor version to 7.3.0

File: bitnami/apache/templates/deployment.yaml (hunk 3 above)
Comment (whall89) [reply bitnami/apache/templates/deployment.yaml]: changed port back to http to use port value declared at line 73

**Pull request merged**